pkg/kexec-tools/Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG EVE_BUILDER_IMAGE=lfedge/eve-alpine:db4459afe60aa8affb415cdc370f568fe8bc99bd-dirty
+FROM ${EVE_BUILDER_IMAGE} AS build
+
+ENV BUILD_PKGS patch curl make gcc perl util-linux-dev git mtools linux-headers musl-dev xz-dev elfutils-dev libbz2
+ENV PKGS xz-libs elfutils-dev libbz2
+
+RUN eve-alpine-deploy.sh
+
+# Build kexec-tools
+WORKDIR /tmp/kexec/kexec-tools-2.0.23
+RUN curl -L https://mirrors.edge.kernel.org/pub/linux/utils/kernel/kexec/kexec-tools-2.0.23.tar.gz | tar -C .. -xzvf -
+RUN ./configure --prefix /usr && \
```

Comment:
I am actually starting to think that perhaps a better design could be to leverage the kernel package that we're building. I believe there were tricks that @deitch or Rolf may remember when it comes to building debug/profile/etc. version of the kernel package.

Reply:
IMO that could be better, indeed. Looks very ugly when each small util or tool spoils the pkg/*.

Reply:
If I understand @rvs correctly, we can build a few dump related tools (kexec-tools, makedumpfiles) from the pkg/kernel/Dockerfile, and not to create a separate package pkg/kexec-tools. I like this idea to put kernel related things to the kernel package.

Reply:
So eventually I did not like an idea to overcomplicate the kernel Dockerfile, which is already quite messy. The other disadvantage of this approach is that we actually have *two* kernels (old,new) which basically have similar Dockerfiles with only few minor differences (like versions of kernels), so any maintenance of these two files keeping them equal is already a pain. I would keep kexec separate, exactly as a kdump pkg